style.css

```diff
@@ -540,3 +546,238 @@ span {
   font-size: 13px;
   font-weight: 600;
 }
+
+.internal-link {
+  display: none;
+}
+
+@media screen and (min-width: 1024px) {
+  body {
+    width: 100%;
+    overflow-x: auto;
+  }
+
+  .contact textarea {
+    font-size: 16px;
+    border-radius: 0;
+    width: 319px;
+    height: 100px;
+  }
+
+  .ending-bar { display: none; }
+
+  .wrap .contact button {
+    align-self: center;
+    padding: 10px;
+  }
```

Comment:
- [ ] As requeted by the previous reviewer, Kindly make sure the submit form button background color is the same as the one one the provided Figma design.

   - ### YOUR DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/221078264-b0db7f03-de8e-42e2-9a72-49be89d02d3c.png)

   - ### FIGMA DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/221078314-60eb988d-ae4e-4a4c-826b-b0a5c2ee05cc.png)

Reply:
Thank you for notifying and helping me to solve this. I have resolved the issue